backend/routes.js

```diff
@@ -82,6 +82,22 @@ route.GET('/eventsBefore/{before}/{limit}', {}, async function (request, h) {
   }
 })
 
+route.GET('/eventsBetween/{startHash}/{endHash}', {}, async function (request, h) {
+  try {
+    const { startHash, endHash } = request.params
+    const { offset } = request.query
```

Comment:
Here is where it should convert `offset` to a `number` if it's defined, or `0` if it isn't, so that `streamEntriesBetween` can expect a type of `number` for `offset`

Reply:
Okay.